src/main_data.cpp

```diff
@@ -93,25 +94,33 @@ void Main_Data::Init() {
 			getcwd(working_dir, 255);
 			project_path = std::string(working_dir);
 #elif defined(__vita__)
-			// Check if app0 filesystem contains the title id reference file
-			FILE* f = fopen("app0:/titleid.txt","r");
-			if (f == NULL)
-				project_path = "ux0:/data/easyrpg-player";
+			// Check if app is invoked with an externalized game path
+			char boot_params[1024];
+			sceAppMgrGetAppParam(boot_params);
+			if (strstr(boot_params,"psgm:play") && strstr(boot_params, "&project_path=")) {
```

Comment:
One minor thing, please use 'project-path' (with hyphen) for consistency with our official arguments.